internal/interpreter/api.rs

```diff
@@ -578,8 +579,18 @@ impl ComponentCompiler {
                 Box<dyn core::future::Future<Output = Option<std::io::Result<String>>>>,
             > + 'static,
     ) {
-        self.config.open_import_fallback =
-            Some(Rc::new(move |path| file_loader_fallback(Path::new(path.as_str()))));
+        let open_import_fallback = move |file_name: String| -> core::pin::Pin<
+            Box<
+                dyn core::future::Future<
+                    Output = Option<std::io::Result<(String, SourceFileVersion)>>,
+                >,
+            >,
+        > {
```

Comment:
I wonder if this works:
```suggestion
        let open_import_fallback = move |file_name: String| {
```

Or at least use `_`  or `use` to make this beast readable.